### PR TITLE
Add multi-instance tt docs, tt clean

### LINKED ISSUE #3130: tt multiinstance applications
**Epic: [Document TT CLI](https://github.com/tarantool/doc/issues/2553)**
Related dev page: https://github.com/tarantool/tt/issues/7

**Product:** Tarantool CLI
**Since:** 2.10
**Audience/target:** dev, admin
**Root document:** https://github.com/tarantool/tt#working-with-a-set-of-instances
**SME:** @ 0x501D 

# Details

Update descriptions of `tt connect/start/stop/status` to describe working with multiinstance applications.
Add a new page for `tt clean`.
All details are in the tt [readme](https://github.com/tarantool/tt#working-with-a-set-of-instances)

### PR DESCRIPTION
**Pull request by p7nov**
Resolves #3130

### CODE REVIEW

**Review by 0x501D — APPROVED** [review]
LGTM

File: doc/reference/tooling/tt_cli/multiple_instances.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Managing multiple instances
+===========================
+
+The ``tt`` utility can manage Tarantool applications that run on
+multiple instances. With a single ``tt`` call, you can:
+
+*   start an application on multiple instances (``tt start``)
+*   check the status of application instances (``tt status``)
+*   connect to a specific instance of an application (``tt connect``)
+*   stop a specific instance of an application or all its instances (``tt stop``)
```
Comment (patiencedaur) [inline doc/reference/tooling/tt_cli/multiple_instances.rst]: Let's provide links to these commands, like so: [tt stop](https://www.tarantool.io/en/doc/latest/reference/tooling/tt_cli/stop/)

File: doc/reference/tooling/tt_cli/multiple_instances.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+This directory should contain the following files:
+
+*   The application file named ``init.lua``.
+*   The instances configuration file ``instances.yml`` with the following format:
+
+    ..  code:: yaml
+
+        <instance_name1>:
+        <instance_name2>:
+        ...
```
Comment (patiencedaur) [inline doc/reference/tooling/tt_cli/multiple_instances.rst:30]: Does this example suggest that there are only instance names in the file and nothing else? I'd imagine something like this:
```
<instance_name1>:
  advertise_uri: 172.19.0.10:3301
  http_port: 8081
```
If I'm wrong and the file only contains instance names with semicolons, please make it clear.

**Review by patiencedaur — COMMENTED** [review]
Great job! Kudos for the alphabetized list.
Please find my comments below.

File: doc/reference/tooling/tt_cli/multiple_instances.rst (hunk 2 above)
Comment (p7nov) [reply doc/reference/tooling/tt_cli/multiple_instances.rst:30]: Only instance names for now.
A follow-up task on documenting available parameters once they are ready: https://github.com/tarantool/doc/issues/3156

**Pull request merged**